simpleRepository/MemoryRepository.test.ts

```diff
@@ -0,0 +1,207 @@
+/**
+ * Unit test for MemoryRepository and hgrs.
+ *
+ * 
+ */
+
+
+import { isStoredRepositoryItem, StoredRepositoryItem, StoredRepositoryItemTestCallback } from "./types/StoredRepositoryItem";
+import { MemoryRepository } from "./MemoryRepository";
+import { RepositoryEntry } from "./types/RepositoryEntry";
+import { isArrayOf } from "../../core/modules/lodash";
```

Comment:
Should this be `import { isArrayOf } from "../modules/lodash";` or am I missing something?

Reply:
Yes, this should be shorter `../modules/lodash`. 

WebStorm/IDEA actually warns about this and it can be fixed with a click.